src/Migrations/Cli/Command/GenerateMigrationCommand.php

```diff
@@ -63,5 +63,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->migrationGenerator->generate($timestamp, $config['migrations']['namespace'], $path);
 
         $this->io->success(sprintf('Generated migration in %s/Version%s.php', $path, $timestamp));
+
+        return 0;
```

Comment:
Can we please use the constant instead? \Symfony\Component\Console\Command\Command::SUCCESS

Reply:
Tested it and it didn't break anything so will replace it

Reply:
Brill, thank you!
Would you please do so for all the Command classes? 
They're under
Migrations/Cli/Command/
Patch/Cli/Command/
Permissions/Cli/Command/
Scripts/Cli/Command

Reply:
Just pushed the changes, shall I mark this as resolved or will you take another look and resolve it yourself?